### PR TITLE
add /nodes API for explorer

### PR DESCRIPTION
**Pull request by lzl124631x**
TODO:
* add real GetNodeCount Logic.
* consider adding `/sync` for syncing. Currently `/blocks` and `/nodes` are separate APIs.

### CODE REVIEW

**Comment by lzl124631x** [comment]
@LeoHChen Do you know why this failed?

File: api/service/explorer/service.go [hunk 1]
```diff
@@ -247,6 +251,15 @@ func (s *Service) GetExplorerAddress(w http.ResponseWriter, r *http.Request) {
 	json.NewEncoder(w).Encode(data.Address)
 }
 
+// GetExplorerNodes serves /nodes end-point.
+func (s *Service) GetExplorerNodes(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+
+	var nodes int
+	nodes = 50 + rand.Int()%50 // TODO(ricl): get nodes count.
```
Comment (LeoHChen) [inline api/service/explorer/service.go]: get the length of consensus.PublicKeys.  You may add a variable in internal/configs/node/config.go, ConfigType, and update it regularly with the start of each consensus.

**Comment by harmony-ek** [comment]
Merged from `master` to fix CI builds.

File: api/service/explorer/service.go [hunk 2]
```diff
@@ -24,19 +24,21 @@ const (
...
-	router      *mux.Router
-	IP          string
-	Port        string
-	storage     *Storage
-	server      *http.Server
-	messageChan chan *msg_pb.Message
+	router            *mux.Router
+	IP                string
+	Port              string
+	getPublicKeyCount func() int
```
Comment (LeoHChen) [inline api/service/explorer/service.go]: better change the function name to getNumPeers

**Pull request merged**